mod/fabric/build.gradle

```diff
@@ -18,13 +18,11 @@ configurations {
 dependencies {
 	modImplementation "net.fabricmc:fabric-loader:${rootProject.fabric_loader_version}"
 	modApi "net.fabricmc.fabric-api:fabric-api:${rootProject.fabric_api_version}"
+	modApi "dev.architectury:architectury-fabric:${rootProject.architectury_version}"
```

Comment:
Dos this require users to have achitectury installed? Or is this purely at build time?

Reply:
i think its embedded

Reply:
It does require users to install [Architectury API](https://modrinth.com/mod/architectury-api). It's a useful API though, it adds [Platform.isDevelopmentEnvironment()](https://docs.architectury.dev/api/platform#platformisdevelopmentenvironment), which means you can hide Mojang authentication behind a dev-environment check.

Reply:
I don't see the value of shipping development flags to production.

Reply:
Do we have another way to reliably assert that it's a dev environment?